utils/helpers.js

```diff
@@ -283,5 +296,6 @@ module.exports = {
     pollForNewERC721HollowWalletOwner,
     pollForNewSignerBalance,
     pollForNewWalletBalance,
-    unPauseAndPoll
+    unPauseAndPoll,
+    pollForEvent: pollForResponseCode
```

Comment:
nit: pollForEvent begs the question which event.
Should we leave it as pollForResponseCode since that's what it does?

Reply:
I renamed it to `pollForEvent` so that it could be more generic and reused.